crates/viewer/re_viewer/src/ui/welcome_screen/welcome_section.rs

```diff
@@ -56,7 +56,9 @@ pub(super) fn welcome_section_ui(ui: &mut egui::Ui) {
         if ui
             .button(
                 egui::RichText::new("Go to documentation →")
-                    .color(egui::Color32::from_hex("#60A0FF").expect("this color is valid"))
+                    .color(
+                        egui::Color32::from_hex("#60A0FF").expect("color failure (internal error)"),
```

Comment:
As noted in other PR: This should be using the `egui::hex_color!` macro instead: https://docs.rs/egui/latest/egui/macro.hex_color.html